src/LibVLCSharp/MediaPlayer.cs

```diff
@@ -3596,4 +3613,35 @@ public enum AudioOutputMixmode
         /// </summary>
         M7_1 = 5
     }
+
+    /// <summary>
+    /// Enumeration of values used to set the video fitting inside the display area.
+    /// </summary>
+    public enum VideoFitMode
+    {
+        /// <summary>
+        /// Explicit zoom set by <see cref="MediaPlayer.Scale"/>
+        /// </summary>
+        FitNone,
```

Comment:
Wouldn't it be better to explicitely set the enum values ?

Reply:
We could. Pinged upstream in case they want to add it.

Reply:
https://code.videolan.org/videolan/vlc/-/merge_requests/5295